docs_user/modules/con_adoption-limitations.adoc

```diff
@@ -29,7 +28,7 @@ The following {compute_service_first_ref} features are Technology Previews:
 * AMD SEV
 * Direct download from Rados Block Device (RBD)
 * File-backed memory
-* `Provider.yaml`
+* Defining custom traits and resource classes in a YAML file
```

Comment:
YAML is just a mark up language, should this specifically state "manifest file"?